modules/Core/src/test/java/org/terasology/world/generator/TreeTests.java

```diff
@@ -126,9 +126,9 @@ public Block setBlock(int x, int y, int z, Block block) {
             };
 
             Random random = new MersenneRandom(seed);
-            BlockManager blockManager = CoreRegistry.get(BlockManager.class);
+            BlockManager blockManagerLocal = CoreRegistry.get(BlockManager.class);
```

Comment:
Only slightly different situation from the biomeManagerLocal comment earlier: in this case here "blockManager" is appropriate, because the one stored in the CoreRegistry is THE blockManager instance. However, the class-level field "blockManager" and "biomeManager" would be better named mockBlockManager and mockBiomeManager because they are mock objects.